tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -74,6 +74,25 @@ namespace EnergyPlus {
 
 		}
 
+		TEST_F(InputProcessorFixture, MisleadingIDDWarningTest)
+		{
+			bool errors_found = false;
+
+			std::string const idf_contents = delimited_string({
+				"Timestep, 4;"
+			});
+
+			ASSERT_FALSE( process_idf( idf_contents, errors_found ) );
+			EXPECT_TRUE( compare_err_stream( "", true ) );
+
+			std::string saveMatchVersion = DataStringGlobals::MatchVersion;
+			DataStringGlobals::MatchVersion = "2.0";
+			ASSERT_FALSE(process_idf(idf_contents, errors_found ));
+			EXPECT_TRUE( compare_err_stream( "   ** Severe  ** IP: Possible incorrect IDD File\r\n   **   ~~~   ** " + DataStringGlobals::IDDVerString +" not the same as expected =\"2.0\"\r\n", true ) );
```

Comment:
@mbadams5 This unit test compares a multi-line err_stream, so there's a line ending problem.  Works on Windows with \r\n but on linux CI it fails, because there's just \n [here](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-d78f8661dece14c04375b0cf38af915499b402be-PR5292-x86_64-Linux-Ubuntu-14.04-gcc-4.8.html).  Any suggestions?  Would you be open to a findin_err_stream function?  Then one could test for specific key content in the error stream without need to build up a long comparison string.